routeros/resource_ip_vrf.go

```diff
@@ -0,0 +1,41 @@
+package routeros
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// ResourceIPRoute https://wiki.mikrotik.com/wiki/Manual:Virtual_Routing_and_Forwarding
+func ResourceIPVrf() *schema.Resource {
+	resSchema := map[string]*schema.Schema{
+		MetaResourcePath: PropResourcePath("/ip/vrf"),
+		MetaId:           PropId(Id),
+
+		"disabled": {
```

Comment:
There is a helper for this attribute and it is better to use it: `KeyDisabled: PropDisabledRw,`